sqlite/src/main.c

```diff
@@ -1549,6 +1569,14 @@ const char *sqlite3ErrStr(int rc){
       if( ALWAYS(rc>=0) && rc<ArraySize(aMsg) && aMsg[rc]!=0 ){
         zErr = aMsg[rc];
       }
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
+      else if( rc>=comdb2MinErr ){
+          rc -= comdb2MinErr;
+          if( rc<ArraySize(aComdb2Msg) && aComdb2Msg[rc]!=0 ){
+            zErr = aComdb2Msg[rc];
+          }
+      }
+#endif
```

Comment:
Minor nit: This `#endif` and the one above it are missing the special comment marker, e.g.:

`#endif /* defined(SQLITE_BUILDING_FOR_COMDB2) */`

Reply:
Yes. Thank you, Joe!